test/cli_spec.js

```diff
@@ -72,22 +74,37 @@ describe(`electron-forge CLI (with installer=${installer.substr(12)})`, () => {
       await pSpawn(['package', dir]);
     });
 
-    let targets = [];
-    if (fs.existsSync(path.resolve(__dirname, `../src/makers/${process.platform}`))) {
-      targets = fs.readdirSync(path.resolve(__dirname, `../src/makers/${process.platform}`)).map(file => path.parse(file).name);
-    }
-    const genericTargets = fs.readdirSync(path.resolve(__dirname, '../src/makers/generic')).map(file => path.parse(file).name);
-
-    [].concat(targets).concat(genericTargets).forEach((target) => {
-      describe(`make (with target=${target})`, () => {
-        before(async () => {
-          const packageJSON = JSON.parse(await fs.readFile(path.resolve(dir, 'package.json'), 'utf8'));
-          packageJSON.config.forge.make_targets[process.platform] = [target];
-          await fs.writeFile(path.resolve(dir, 'package.json'), JSON.stringify(packageJSON));
-        });
+    it('can package without errors', async () => {
+      await pSpawn(['package', dir]);
+    });
 
-        it('successfully makes with default config', async () => {
-          await pSpawn(['make', dir, '-s']);
+    it('can package without errors with native pre-gyp deps installed', async () => {
+      await installDeps(dir, ['ref']);
+      await pSpawn(['package', dir]);
+    });
+
+    describe('after package', () => {
+      before(async () => {
+        await pSpawn(['package', dir]);
+      });
+
+      let targets = [];
+      if (fs.existsSync(path.resolve(__dirname, `../src/makers/${process.platform}`))) {
+        targets = fs.readdirSync(path.resolve(__dirname, `../src/makers/${process.platform}`)).map(file => path.parse(file).name);
+      }
+      const genericTargets = fs.readdirSync(path.resolve(__dirname, '../src/makers/generic')).map(file => path.parse(file).name);
+
+      [].concat(targets).concat(genericTargets).forEach((target) => {
+        describe(`make (with target=${target})`, () => {
+          before(async () => {
+            const packageJSON = JSON.parse(await fs.readFile(path.resolve(dir, 'package.json'), 'utf8'));
```

Comment:
I feel like this line is repeated a bunch of times in the codebase, but that's a refactor for another PR.